cli/src/program.rs

```diff
@@ -279,6 +283,14 @@ impl ProgramSubCommands for App<'_, '_> {
                                 .long("no-auto-extend")
                                 .takes_value(false)
                                 .help("Don't automatically extend the program's data account size"),
+                        )
+                        .arg(
+                            Arg::with_name("skip_feature_verify")
+                                .long("skip-feature-verify")
+                                .takes_value(false)
+                                .help("Don't verify program against the activated feature set. \
+                                This setting means a program containing a syscall not yet active on \
+                                mainnet will not fail verification.")
```

Comment:
nit: maybe we can add a bit more info
```suggestion
                                mainnet will succeed local verification, but fail during the last step of deployment.")
```